### DISCUSSION TITLE
Notification count (+do not disturb red dot?)

### DISCUSSION THREAD

**Discussion by andreyrd** [lawand-dot-io/taskbar#55] [Feature requests]
I'd like to see the actual notification count. That might be hard to achieve with the current size taskbar, but once taskbar resizing is implemented we can have a taller taskbar with bigger icons that can fit an actual notification count.

**Comment by lawand-dot-io** [comment]
Actually this might be doable without increasing the size of Taskbar. Other apps (such as uBar) already do this by showing a notification icon that's the same size as the app icon. I will consider doing this then.

**Comment by lawand-dot-io** [comment]
Split the "Show notifications for app icons (added using the "Keep in Taskbar" option)" feature request into https://github.com/lawand-dot-io/taskbar/discussions/75

**Comment by lawand-dot-io** [comment]
Split the "Show running indicator for app icons (added using the "Keep in Taskbar" option)" feature request into https://github.com/lawand-dot-io/taskbar/discussions/76

**Comment by lawand-dot-io** [comment]
Update: can also try supporting the "Do not disturb" mode for notifications as mentioned here: https://github.com/lawand-dot-io/taskbar/issues/115
